README.md

```diff
@@ -1,5 +1,19 @@
-# Django ORM boilerplate for GitHub tasks
+# Taxi service search and tests
 
-- Warning: Use `pytest app` for testing - not simple `pytest`
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
\ No newline at end of file
+
+In this task, you will implement search feature for your website and write tests for your project.
+
+1. Implement search feature for all 3 pages with content:
+   - drivers - by username
+   - cars - by model
+   - manufacturers - by name
+2. Write tests for custom and for core project features (no need to test built-in functionality).
+   Use this [tutorial](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Testing) as helper.
+   You decide what to test and how to test (train your testing skills here).
+3. Make your website looks better. Customize your templates, add static images for taxi service.
+   Just make this website looks better use your imagination :)
```

Comment:
you use `make your/this website looks better` twice in one line. Sounds a bit strange